// Copyright 2023 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//    https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package guideline.vision;

// Options for the DepthAlignmentCalculator
message DepthAlignmentCalculatorOptions {
  // Whether the input image represents inverse depth (false for metric depth).
  bool input_inverse_depth = 1;
  // Whether the output image should be inverse depth (false for metric depth).
  bool output_inverse_depth = 2;
}
